### ISSUE TITLE
using syringe on hypospray doesnt toggle inject/draw

### ISSUE THREAD

**Issue by deltanedas** [RMC-14/RMC-14#4358]
## Description
if you inject it stays on inject while empty, if you draw it stays on draw while full

**Comment by Vermidia** [comment]
Yeah this is an annoying one since all injector logic basically had to be duplicated for the hypo code, so it would need to check at the end to switch modes too.
